packages/@okta/vuepress-site/docs/guides/oauth-for-okta-apis/define-allowed-scopes/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Define allowed scopes
+---
+When a request is sent to the Okta Org Authorization Server, it validates all of the requested scopes in the access token against the app's grants collection. The scope is granted if it exists in the app's grants collection.
```

Comment:
this should be something like "validates all of the scopes in the OAuth2 authorize or token request". 

Requested scopes are not in an access token. Only granted scopes are in an access token.